analysis/monkwindwalker/src/modules/covenants/FaelineStomp.tsx

```diff
@@ -0,0 +1,82 @@
+import { formatNumber, formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import { SpellIcon } from 'interface';
+import Analyzer, { Options, SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import calculateEffectiveDamage from 'parser/core/calculateEffectiveDamage';
+import Events, { DamageEvent } from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import React from 'react';
+import Enemies from 'parser/shared/modules/Enemies';
+
+
+
+const MOD = 0.08;
+
+class FaelineStompWindwalker extends Analyzer {
```

Comment:
Shouldn't this simply be called FaelineStomp? We're already in the windwalker directory and then it matches with the file naming

Reply:
Changed the naming since it was a bit misleading. Its not the stomp this module is about and more the dmg from the legendary. So both modules, the original nf ww and the new one should exist at the same time.

Reply:
In that case it should be called FaelineHarmony to match the legendary?

Reply:
Yes, renamed it already but put 'dmg' at the end to make clear this is only for the dmg calculation (in case abe wants to make a module for the calculation of the healinggain through this legendary)

Reply:
I mean it's already under windwalker, so if there's any healing for the windwalker it should just be included in this file and then rendered alongside the damage in a <ItemHealingDone/>